docs/user/README.md

```diff
@@ -252,6 +252,18 @@ By using Telegraf, data points can be efficiently streamed into `modelardbd` fro
 [collection of data sources](https://www.influxdata.com/time-series-platform/telegraf/telegraf-input-plugin/)
 such as [MQTT](https://mqtt.org/) and [OPC-UA](https://opcfoundation.org/about/opc-technologies/opc-ua/).
 
+## ModelarDB configuration
+`ModelarDB` can be configured before the server is started using environment variables. A full list of the environment 
+variables is provided here. If an environment variable is not set, the specified default value will be used.
+
+| **Variable**                                     | **Default** | **Description**                                                                                                                                                                                                                                                                                                                                            |
+|--------------------------------------------------|-------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| MODELARDBD_PORT                                  | 9999        | The port of the Apache Arrow Flight Server.                                                                                                                                                                                                                                                                                                                |
+| MODELARDBD_UNCOMPRESSED_RESERVED_MEMORY_IN_BYTES | 512 MB      | Amount of memory to reserve for storing uncompressed data buffers.                                                                                                                                                                                                                                                                                         |
```

Comment:
Maybe prefix with "The" to match the other settings.

Reply:
Fixed.